### PR TITLE
Tweak inverse_kinematics

### PR DESCRIPTION
**Pull request by corrigentia**
Fix some grammar, punctuation, word order, word choice and GDScript style guide compliance in inverse_kinematics.

### CODE REVIEW

File: tutorials/3d/inverse_kinematics.rst [hunk 1]
```diff
@@ -17,18 +17,18 @@ move each joint in our arm to get to that target.
 Initial problem
 ~~~~~~~~~~~~~~~
 
-Talking in Godot terminology, the task we want to solve here is to position
+In Godot parlance, the task we want to solve here is to position
```
Comment (clayjohn) [inline tutorials/3d/inverse_kinematics.rst]: We should remove the "In Godot parlance", it doesn't make sense in context and "parlance" is not a very accessible word for many people. How about instead we just use: "We want to position the 2 angles..."

File: tutorials/3d/inverse_kinematics.rst [hunk 2]
```diff
@@ -17,18 +17,18 @@ move each joint in our arm to get to that target.
...
 lowerarm bone is as close to the target point (which is set by the target Vector3)
 as possible using only rotations. This task is calculation-intensive and never
 resolved by analytical equation solving, as it is an under-constrained
-problem which means that there is more than one solution to an
+problem, which means that there is more than one solution to an
 IK problem.
 
 .. image:: img/inverse_kinematics.png
 
 For easy calculation in this chapter, we consider the target being a
```
Comment (clayjohn) [inline tutorials/3d/inverse_kinematics.rst]: I know you didn't touch this line yourself. But we should change it to, "we assume the target is a child of Skeleton".

File: tutorials/3d/inverse_kinematics.rst (hunk 1 above)
Comment (corrigentia) [reply tutorials/3d/inverse_kinematics.rst]: > We should remove the "In Godot parlance", [...] "parlance" is not a very accessible word for many people. How about instead we just use: "We want to position the 2 angles..."

@clayjohn : Amended.

> "In Godot parlance", [...] doesn't make sense in context

I had rephrased the awkward wording intended to point out that that was not an everyday way of looking at such a task, but a specialized, technical, algorithmic manner of breaking down a problem into steps and procedures that can be carried out by a computer agent.

File: tutorials/3d/inverse_kinematics.rst (hunk 2 above)
Comment (clayjohn) [reply tutorials/3d/inverse_kinematics.rst]: My bad, I seem to have commented below the wrong line. This change was intended for line 30. Which currently says "For easy calculation in this chapter, we consider the target being a child of Skeleton."

We should change it to "For easy calculation in this chapter, we assume the target is a child of Skeleton."

Comment (corrigentia) [reply tutorials/3d/inverse_kinematics.rst]: Much obliged, @clayjohn.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**